code/modules/projectiles/guns/syringe_gun.dm

```diff
@@ -315,7 +315,7 @@
 	if(!S)
 		return
 
-	chambered.BB = new S.projectile_type(src)
+	chambered.BB = new /obj/item/projectile/bullet/dart/syringe/pierce_ignore(src)
```

Comment:
why not change `projecitle_type` to `/obj/item/projectile/bullet/dart/syringe/pierce_ignore`

Reply:
nvm I see why now.